### ISSUE TITLE
Questing regarding key rebinding

### ISSUE THREAD

**Issue by qutetemp** [qutebrowser/qutebrowser#2978]
I don't really know what I'm missing. I want to bind tab-close and undo to x respectively X.
This should do the trick right ? Isn't this supposed to free d and u ? Should I config.unbind(',d', mode='normal') first "

config.bind(',x', 'tab-close', mode='normal', force=True)
config.bind(',X', 'undo', mode='normal', force=True)

**Comment by jgkamat** [comment]
`config.bind` will not clear the previous binding. I also think that the `,` before your `X` might be tripping you up.

I think this is what you're looking for:

```
config.bind('x', 'tab-close', mode='normal', force=True)
config.bind('X', 'undo', mode='normal', force=True)
config.unbind('d', mode='normal')
config.unbind('u', mode='normal')
```

You might also want to put a `config.load_autoconfig = False` into your config file, I'm having a few problems with my bindings being overwritten when I don't have that in (right now).

**Comment by qutetemp** [comment]
It works now, thanks.I used that comma because it was like that in the documentation.

**Comment by The-Compiler** [comment]
The example is like that because comma is a good leader key for custom bindings - the newest version mentions:

> Key chains starting with a comma are ideal for custom bindings, as the comma key will never be used in a default keybinding.

@qutetemp Was that already in when you read the docs? if not, would that have made things clearer?

**Comment by qutetemp** [comment]
I am unsure if the comma really was the problem and judging by your message it probably wasn't. I just felt the need to state why I used it because I missed what you quoted and I thought that was the problem.

I nearly finished my new config. In the end I used this:

config.bind('X', 'undo', mode='normal', force=True)
config.bind('x', 'tab-close', mode='normal', force=True)
config.bind('K', 'tab-next', mode='normal', force=True)
config.bind('J', 'tab-prev', mode='normal', force=True)
config.bind('d', 'scroll-page 0 0.5', mode='normal', force=True)
config.bind('u', 'scroll-page 0 -0.5', mode='normal', force=True)
